Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -24,7 +24,6 @@
         generate_parser_c_extension,
         generate_c_parser_source,
     )
-    from pegen.ast_dump import ast_dump
```

Comment:
Looks like this is used by various strings (templates) that are run via `self.run_test()`.

Reply:
Interesting, no tests appear to fail if this import is removed. (I checked.)

Reply:
Oh, you're right. The template also imports this, L40. Nvm then.